src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+export const PostList: React.FC<{ posts: Post[] }> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
don't forget about `key` by mapping the list 
```suggestion
      <PostInfo post={post} key={post.id} />
```